docs/api/usb/USBCDC_ECM.md

```diff
@@ -0,0 +1,52 @@
+# USBCDC_ECM
+
+<span class="images">![](https://os.mbed.com/docs/mbed-os/development/mbed-os-api-doxy/class_u_s_b_c_d_c___e_c_m.png)<span>USBCDC_ECM class hierarchy</span></span>
+
+The USBCDC_ECM class emulates an Ethernet interface over USB. You can use the interface to send and receive Ethernet frames over USB.
```

Comment:
Query: What are the typical use cases for this class? What are the common scenarios in which you'd use it?

Reply:
Typical applications are USB to Ethernet adapter and USB Web server that allows users to configure a USB device and to view data using a web browser.  The user needs to add required SW components on top of the USBCDC_ECM class like IP stack and Web server.